src/TransactionExecutable.ts

```diff
@@ -35,5 +34,5 @@ export interface TransactionExecutable {
      *                   filling in parameters of the statement.
      * @returns Promise which fulfills with a Readable.
```

Comment:
Update docstring

Reply:
I kept it Readable on purpose. `ResultReadable` is extending Readable, and most customers might just need Readable, I think `Promise which fulfills with a Readable` would be more helpful than `ResultReadable`.